pkg/topology/mock/mock.go

```diff
@@ -16,14 +16,14 @@ type mock struct {
 	peers           []swarm.Address
 	closestPeer     swarm.Address
 	closestPeerErr  error
-	addPeerErr      error
+	AddPeersErr     error
```

Comment:
It looks to me that this does not have to be exported.

Reply:
Ah, that was a rename. This is the last time I used rename all functionality, I am apparently very bad at using it :)